packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -861,7 +870,7 @@ export class StripeHelper {
     customerId: string;
     postalCode: string;
     country: string;
-  }): Promise<boolean> {
+  }): Promise<void> {
```

Comment:
It's not useful to know whether the location was actually set anymore?

Reply:
We changed the calling code around a bit and ended up not using the return value.  However, we should re-throw the exception so the calling code will log it.

Reply:
We added the return value back in, as it turns out we need it for the CustomerLocations call site.